Samples/Target group condition Login status/Resources/LoginStatusTargetGroupCondition.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Litium.Foundation.Security;
+using Litium.Web.Customers.TargetGroups;
+using Litium.Web.Customers.TargetGroups.Events;
+using System.Collections.Concurrent;
+
+namespace Litium.Accelerator.TargetGroupconditions.LoginStatus
+{
+    /// <summary>
+    /// The TargetGroupCondition is responsible for parsing the condition when a user visits the page.
+    /// It contains the logic to check if the current user matches the defined target group condition.
+    /// </summary>
+    public class LoginStatusTargetGroupCondition : ITargetGroupCondition<LoginStatusFilterCondition>
+    {
+        private LoginStatusFilterCondition _data;
+
+        /// <summary>
+        /// If set to true the condition support both Add and Remove as actions so that a user can be removed 
+        /// from a condition where previously matched.
+        /// </summary>
+        public bool AllowsMultipleActions => true;
+
+        /// <summary>
+        /// Called to check which action that should be executed if the process action returns True.
+        /// This method is called AFTER the call to Process and is only called if a change is to be made (if Proces() returns true)
+        /// </summary>
+        public TargetGroupProcessAction GetProcessAction(ConcurrentDictionary<string, object> context, ITargetGroupEvent targetGroupEvent)
+        {
+            var result = TargetGroupProcessAction.Add;
+            bool isLoggedIn = IsUserLoggedIn(targetGroupEvent);
+            if (_data.Value == LoginStatusFilterService.ValueAnonymous)
+            {
+                // User be member of anonymous condition on only if not logged in, otherwise removed
```

Comment:
Should this be _User should be member of anonymous condition only if not logged in, otherwise removed_?